test/karma.conf.js

```diff
@@ -25,19 +30,50 @@ module.exports = function(config) {
       'dist-test/webpack-test.js',
       'dist-test/videojs-http-streaming.js'
     ],
+    browserStack: {
+      project: 'videojs-http-streaming',
+      name: 'videojs-http-streaming' + process.env.TRAVIS_BUILD_NUMBER,
```

Comment:
I think it would be more useful to have `TRAVIS_BUILD_NUMBER + ' ' + TRAVIS_BRANCH`, testing it [here](https://github.com/videojs/video.js/pull/4903/files) currently.
We should also add in `build` so it's easier to quickly see what build is what. The project name is already easily visible so, not sure it's necessary here.